osu.Game/Screens/OnlinePlay/Multiplayer/Participants/ParticipantPanel.cs

```diff
@@ -162,6 +162,15 @@ protected override void OnRoomUpdated()
 
             const double fade_time = 50;
 
+            var ruleset = rulesets.GetRuleset(Room.Settings.RulesetID);
+
+            var currentModeRank = User.User?.GetStatisticsFor(ruleset)?.GlobalRank;
+
+            // fallback to current mode rank for testing purposes.
+            currentModeRank ??= User.User?.CurrentModeRank;
+
+            userRankText.Text = currentModeRank != null ? $"#{currentModeRank.Value:N0}" : string.Empty;
```

Comment:
Not sure about this change (addition of `N0` at the end), previously it wasn't using commas in-between digits, but felt somewhat wrong, here's a *real-life* comparsion for reference:

Without commas:
![image](https://user-images.githubusercontent.com/22781491/107267127-85895100-6a57-11eb-9e25-5bd995fbe452.png)

With commas:
![image](https://user-images.githubusercontent.com/22781491/107267181-976af400-6a57-11eb-850a-8cdbce7617a1.png)

Reply:
fyi `FormatUtils.FormatRank()` exists.

Reply:
That method is used for score ranks, not user global ranks. Don't think it's fine displaying something like `1k` or `1.5m`.